test/person_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../person'
+RSpec.describe Person do
+  context 'When initializing a person' do
+    it 'should set the person\'s age correctly' do
+      new_person = Person.new(24, 'Atif', true)
+      expect(new_person.age).to eq(24)
+    end
+    it 'should set the person\'s name correctly' do
+      new_person = Person.new(12, 'Khan Zada', false)
+      expect(new_person.name).to eq('Khan Zada')
+    end
+  end
+end
```

Comment:
- [ ] It would be nice if you follow the project requirements 👍 and make tests for all the methods in the person.rb class to make your app tested properly 😊 You are missing the following methods ⬇️

1. correct_name
2. can_use_service?